hack/test-cri.sh

```diff
@@ -60,7 +60,7 @@ if [ ! -x "$(command -v containerd)" ]; then
   exit 1
 fi
 sudo pkill containerd
-sudo containerd &> ${REPORT_DIR}/containerd.log &
+sudo containerd -l debug &> ${REPORT_DIR}/containerd.log &
```

Comment:
We can enable it in our test, but should not let user add the flag by default when following our dobument.